spec/views/student/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe 'students/edit.html.erb', type: :view do
+  describe 'populate page' do
+    before do
+      @student = Student.create(first_name: 'Joe',
+                                last_name: 'Lally')
+      render
+    end
+
+    describe 'presence of title' do
+      it 'checks for title' do
```

Comment:
you're describing what the test is doing, but you should be describing the functionality of the page.

`it 'shows the name` do'